app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -1,22 +1,75 @@
-import type { ChangeEvent } from 'react';
+import { PageHeading } from '@18f/identity-components';
 import { t } from '@18f/identity-i18n';
+import { formatHTML } from '@18f/identity-react-i18n';
 import { FormStepsButton } from '@18f/identity-form-steps';
 import type { FormStepComponentProps } from '@18f/identity-form-steps';
-import type { VerifyFlowValues } from '../../verify-flow';
+import type { VerifyFlowValues } from '../..';
+import { Accordion } from '@18f/identity-components';
+import parsePhoneNumber from 'libphonenumber-js';
+import type { ChangeEvent } from 'react';
+
+interface PasswordConfirmStepProps extends FormStepComponentProps<VerifyFlowValues> {}
+
+function getDateFormat(date: string | number | Date) {
+  date = new Date(date);
+  const options = { year: 'numeric', month: 'long', day: 'numeric' };
+  return date.toLocaleDateString('en-US', options);
+}
 
-interface PasswordConfirmStepStepProps extends FormStepComponentProps<VerifyFlowValues> {}
+function PersonalInfoSummary(value) {
```

Comment:
Ideally there should be one component per file. Since we already have the folder for step-specific implementation, can we move this to `app/javascript/packages/verify-flow/steps/password-confirm/personal-info-summary.tsx` ?